angular-app/src/app/workspace/workspace.component.ts

```diff
@@ -78,16 +73,33 @@ export class WorkspaceComponent implements OnInit {
       });
     }
 
+    randomColor(): String {
+
+        let colors: string[] = ['#FF2D00', '#001FFF', '#009688']; // red, blue, green
+
+        var randNum = Math.floor(Math.random() * Math.floor(3));
+
+        return colors[randNum];
+
+    }
+
+    getAudioName(file) {
```

Comment:
This function seems to be setting something... should it be called `setAudioFileName`?

Reply:
Yes, it should.